appdmg.json

```diff
@@ -0,0 +1,24 @@
+{
+    "title": "ZimwriterFS Binary",
+    "icon": "openzim.png",
+    "window": {
+        "size": {
+            "width": 640,
+            "height": 344
+        }
+    },
```

Comment:
zimwriterfs is a console application, what the `window` is for ?
(same question for the `x` and `y` above)